src/panels/config/lovelace/resources/dialog-lovelace-resource-detail.ts

```diff
@@ -60,27 +65,34 @@ export class DialogLovelaceResourceDetail extends LitElement {
     fireEvent(this, "dialog-closed", { dialog: this.localName });
   }
 
+  private _closeDialog(): void {
+    this._dialog?.close();
```

Comment:
Do you call this so the animation is handled?
Use the `open` attribute of `ha-dialog-new` instead